store/src/heartbeat.rs

```diff
@@ -57,3 +77,48 @@ impl From<CellHeartbeat> for CellHeartbeatReqV1 {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use prost::Message;
+
+    fn bytes() -> Vec<u8> {
+        hex::decode("008f23e96ab6bbff48c8923cac831dc97111bcf33dba9f5a8539c00f9d93551af1").unwrap()
```

Comment:
Try using `https://docs.rs/hex-literal/0.3.4/hex_literal/` instead.. it allows for const hex declarations

Reply:
Fixed fe126ca